src/python/CMSSpark/reports/aggregate_campaign_tier.py

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env python
+"""
+Spark script to parse and aggregate DBS and PhEDEx records on HDFS.
+"""
+
+# system modules
+import os
+import re
+import sys
+import time
+import json
+import argparse
+from types import NoneType
+
+from pyspark import SparkContext, StorageLevel
+from pyspark.sql import HiveContext
+from pyspark.sql.functions import udf
+
+# CMSSpark modules
+from CMSSpark.spark_utils import dbs_tables, phedex_tables, print_rows
+from CMSSpark.spark_utils import spark_context, split_dataset
+from CMSSpark.utils import elapsed_time
```

Comment:
replace this with
from CMSSpark.utils import info
from CMSSpark.conf import OptionParser